lesson_09/homework_09.py

```diff
@@ -3,4 +3,22 @@
 Створіть об'єкт цього класу, представляючи студента. Потім додайте метод до класу "Студент",
 який дозволяє змінювати середній бал студента. Виведіть інформацію про студента та змініть
 його середній бал.
-"""
\ No newline at end of file
+"""
+class Student:
+    def __init__(self,first_name, last_name, age, gpa):
+        self.first_name = first_name
+        self.last_name = last_name
+        self.age = age
+        self.gpa = gpa
+ 
+    def change_gpa(self, new_gpa):
+        self.gpa = new_gpa
```

Comment:
```suggestion
        self.gpa = new_gpa
```
непогано булоб додати якісь перевірки